web-client/src/components/Button/styled.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+interface IButtonComponent{
+    hooverBackgroundColor?: string;
+}
+const ButtonComponent = styled.button<IButtonComponent>`
+    background-color: white;
+    width: 40px;
+    height: 40px;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 4px;
+    border: none;
+    margin-left: 16px;
+    &:hover{
+        cursor: pointer;
+        background-color: ${({hooverBackgroundColor}) => hooverBackgroundColor ? hooverBackgroundColor : "transparent"}; 
+        border: 2px solid #1976d2;
```

Comment:
Do you think it is possible to use https://mui.com/joy-ui/main-features/global-variants/#customizing-global-variants for colors?

Reply:
This question needs  to research, idea is great. I'll get to it in the near future,thanks!